modules/core/src/lib/constants.js

```diff
@@ -42,3 +43,13 @@ export const COORDINATE_SYSTEM = {
   // Positions and distances are not transformed: [x, y, z] in unit coordinates
   IDENTITY: 0.0
 };
+
+export const SHADER_COORDINATE_SYSTEM = {
+  LNGLAT: COORDINATE_SYSTEM.LNGLAT,
+  LNGLAT_EXPERIMENTAL: COORDINATE_SYSTEM.LNGLAT_EXPERIMENTAL,
```

Comment:
These are not end user visible, so here we could use more descriptive names, such `LNGLAT_HYBRID` vs. `LNGLAT_ORIGINAL`.